### PR TITLE
Enable some support for global keywords

### PR DESCRIPTION
**Pull request by joakim-hove**
This PR adds *limited* support for storing/handling all the cells in 3D field. The support is specifically aimed at addressing a situation which arises with a combination of MULTZ and PINCH:ALL; it is not fully general and does in particular not support all operations on 3D keywords.

~~Test case: https://github.com/OPM/opm-tests/pull/356~~
~~Regression test: https://github.com/OPM/opm-simulators/pull/2765~~

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
jenkins build this opm-tests=356 opm-simulators=2765 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2765 please

**Comment by joakim-hove** [comment]
jenkins build this opm-simulators=2765 please

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Comments to this?!

File: src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp [hunk 1]
```diff
@@ -987,8 +1058,14 @@ void FieldProps::scanREGIONSSection(const REGIONSSection& regions_section) {
...
         const std::string& name = keyword.name();
-        if (keywords::REGIONS::int_keywords.count(name) == 1 || keywords::isFipxxx(name)) {
-            this->handle_int_keyword(keyword, box);
+        if (keywords::REGIONS::int_keywords.count(name)) {
+            this->handle_int_keyword(keywords::REGIONS::int_keywords.at(name), keyword, box);
+            continue;
+        }
+
+        if (keywords::isFipxxx(name)) {
+            auto kw_info = keywords::keyword_info<int>{.scalar_init = 1};
```
Comment (bska) [inline src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp:1067]: You can't use designated initializers.

**Review by bska — COMMENTED** [review]
Looks okay for the most part.  I saw one use of a designated initializer and that must be amended, but other than that I think we should merge this.

File: src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp (hunk 1 above)
Comment (joakim-hove) [reply src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp]: Thank's for pointing out - it has popped up again through a rebase probably; fixed now.

**Comment by joakim-hove** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks for the updates.  I'll merge this into master now.

**Pull request merged**